### DISCUSSION TITLE
selectOnMove for useRadioStore

### DISCUSSION THREAD

**Discussion by crisshaker** [ariakit/ariakit#2674] [Questions]
Is there a reason why there's no selectOnMove for `useRadioStore` like there is for `useSelectStore`?
<img width="412" alt="image" src="https://github.com/ariakit/ariakit/assets/36159205/81dbf9dc-3fa0-471b-a902-55670c2c9061">

It's interesting because that's how radios work natively.

**Comment by diegohaz** [comment]
If you're rendering Radio as a non native input, you'll have to implement this yourself on the radio focus event.

But feel free to submit a feature request with your use case. We can make this easier.
